app/src/main/res/layout/activity_camera.xml

```diff
@@ -1,31 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  android:orientation="vertical"
-  android:gravity="center_horizontal"
-  tools:context=".scrolling.CameraActivity">
-
-  <ImageView
-    android:id="@+id/image_view"
-    android:scaleType="centerCrop"
-    app:srcCompat="@android:drawable/ic_menu_camera"
-    android:layout_width="400dp"
-    android:layout_height="400dp"/>
-
-  <Button
-    android:id="@+id/button_camera"
-    android:text="Camera"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content" />
+    android:layout_height="match_parent"
+    android:gravity="center_horizontal"
+    android:orientation="vertical"
+    tools:context=".CameraActivity">
 
-  <Button
-    android:id="@+id/button_gallery"
-    android:text="Gallery"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content" />
+    <ImageView
+        android:id="@+id/image_Camera_imageView"
+        android:layout_width="400dp"
+        android:layout_height="400dp"
+        android:scaleType="centerCrop"
+        app:srcCompat="@android:drawable/ic_menu_camera" />
+
+    <Button
+        android:id="@+id/camera_Camera_button"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Camera" />
```

Comment:
There are some hardcoded strings in this layout, we should use the @string resources instead.